scripts/mail-alarm

```diff
@@ -388,6 +505,12 @@ class XapiMessage:
         except:
             self.body      = ""
         self.pool_name = get_pool_name()
+        try:
+            with open('/etc/xapi.d/mail-languages/' + mail_language + '.json', 'r') as fileh:
+                self.ml = json.loads(fileh.read())
+        except:
+            log_err("Read mail language pack [\"%s\"] error" % ('/etc/xapi.d/mail-languages/' + mail_language + ".json"))
+            self.ml = None
```

Comment:
I think `self.lang` or even `self.mail_language` is more explicative that `self.ml`